src/System.Net.Security/tests/FunctionalTests/SslStreamAlpnTests.cs

```diff
@@ -87,7 +87,6 @@ public void SslStream_StreamToStream_DuplicateOptions_Throws()
         }
 
         [Theory]
-        [ActiveIssue(24722, TestPlatforms.Linux)]
```

Comment:
This test is failing in the latest runs.

Reply:
Re added a new issue, because the tests are now "fixed" (await for the result) it shows that the code isn't passing at all. So they are disabled.

Reply:
> So they are disabled.

This test is still failing.  Did you mean to add a different [ActiveIssue] for it?